.github/workflows/integration.yml

```diff
@@ -0,0 +1,54 @@
+name: Integration
```

Comment:
Clicking around under the Actions tab, I see we have 3 workflows, lint, spellcheck and wheelbuilder. Where does this integration one go? Is it being run currently?